Submitted By: Alexander E. Patrakov (semzx at newmail dot ru)
Date: 2003-08-13
Initial Package Version: 3.1.3
Origin: Alexander E. Patrakov
Description: Allows the user to copy text from kwrited message window 
and clear that window. Fixes a FIXME.
Applies to only those people who have the "utempter" program.

diff -ur kdebase-3.1.3/konsole/konsole/kwrited.cpp kdebase-3.1.3.tweaked/konsole/konsole/kwrited.cpp
--- kdebase-3.1.3/konsole/konsole/kwrited.cpp	2003-05-07 17:44:16.000000000 +0600
+++ kdebase-3.1.3.tweaked/konsole/konsole/kwrited.cpp	2003-07-30 20:02:42.000000000 +0600
@@ -41,7 +41,7 @@
   wid->setFont(KGlobalSettings::fixedFont());
   wid->setMinimumWidth(wid->fontMetrics().maxWidth()*80 +
       wid->minimumSizeHint().width());
-  wid->setReadOnly(true);
+  wid->setReadOnly(false);
   wid->setFocusPolicy(QWidget::NoFocus);
 
   pty = new TEPty();
